src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.service.impl.ProcessServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriteServiceImpl;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) throws FileNotFoundException {
+        File file = new File("src/main/resources/file.csv");
+        File toFileName = new File("src/main/resources/report.csv");
+        Map<String, Integer> fruitMap = new HashMap<>();
+        ReaderServiceImpl readerService = new ReaderServiceImpl();
+        ProcessServiceImpl processService = new ProcessServiceImpl();
+        ReportServiceImpl reportService = new ReportServiceImpl();
+        WriteServiceImpl writeService = new WriteServiceImpl();
```

Comment:
lets move initialization to class fields or static block and only call service methods in main()

Reply:
fixed